### DISCUSSION TITLE
Translation

### DISCUSSION THREAD

**Discussion by LaBicHetteI** [lllyasviel/Fooocus#3198] [Q&A] [answered]
Hello, this sentence containing (") prevents translations via JavaScript. Can it be modified?
![IMG_9495](https://github.com/lllyasviel/Fooocus/assets/173731046/322380f1-2f71-4c58-8499-9d6d958a3fa8)

**Comment by mashb1t** [comment, marked as answer]
Are you correctly escaping doublequotes with `\"` in your translation file?

**Reply by LaBicHetteI** [reply]
It's OK thanks ;)
